aws-s3-bucket/bucket.tf

```diff
@@ -85,13 +85,13 @@ resource "aws_s3_bucket_cors_configuration" "cors" {
 }
 
 resource "aws_s3_bucket_acl" "acl" {
+  count = var.acl_enabled ? 1 : 0
```

Comment:
Nešlo by se zeptat rovnou na var.acl, ať tam nemusí být další proměnná?

Reply:
slo, ale chtel jsem si pohlidat, aby tam default byl ten "private", kdyby to nekdo pouzil blbe 😅 klidne predelam

Reply:
🤔 aha, tak jo 👍

Reply:
Tak jo, jakoze jsem to mel nechat jak to bylo - nebo tak jo jakoze dobry ze jsem to zmenil, ale chapes muj pristup? 😆

Reply:
Jako že jsi to mohl nechat. 😄 Ale nemám na to silný názor, takže obojí je fajn. 🙂